src/sas/qtgui/MainWindow/GuiManager.py

```diff
@@ -100,9 +103,12 @@ def __init__(self, parent=None):
         handler.messageWritten.connect(self.appendLog)
 
         # Log the start of the session
-        logging.info(" --- SasView session started ---")
+        version = f"{SASVIEW_VERSION.split('.')[0]}.{SASVIEW_VERSION.split('.')[1]}"
```

Comment:
This only gives the major and minor version information, but not which point release is being used (5.0 vs 5.0.5). The full version is given in the next line, but this seems unnecessary.

Perhaps combine the two lines:
`logging.info(f" --- SasView session started: v{SASVIEW_VERSION}, {SASVIEW_RELEASE_DATE} ---")`

Reply:
Adding something like this to MainWindow.py somewhere in `__init__.py` should do the trick
```
        from sas.sasview import __version__ as SASVIEW_VERSION
        self.setWindowTitle("SasView " + str(SASVIEW_VERSION))
```

Reply:
it does.

![Untitled](https://user-images.githubusercontent.com/8266281/130467337-03c8ec59-2793-4123-8ebe-39e3f76849fc.png)